events/app/views/pages/event.njk

```diff
@@ -16,65 +16,94 @@
 <script type="application/ld+json">
   {{ event | jsonLd('eventLd') | safe }}
 </script>
+<div class="row">
+  <div class="container {{ {s:2} | spacingClasses({margin: ['top']}) }}">
+    {% componentV2 'tags', tags %}
+  </div>
+</div>
 
-<article class="container">
-  {# Event title #}
-  <p>{{ event.title }}</p>
-
-  {# Tags #}
-  {% componentV2 'tags', tags %}
-
-  {# Format #}
-  <p>{{ event.format.title }}</p>
-
-  {# Access information #}
-  {% if (event.accessOptions.size !== 0) %}
-    {% for accessOption in event.accessOptions %}
-      {% icon 'a11y/' + (accessOption.accessOption.acronym | lower), accessOption.accessOption.title %}
-    {% endfor %}
-  {% endif %}
+<div class="row">
+  <div>
+    {% if event.promo.image %}
+      {% componentV2 'image', event.promo.image, [], { lazyload: true } %}
+    {% endif %}
+  </div>
+  <div class="container relative {{ {s: -4} | spacingClasses({margin: ['top']}) }}">
+    <div class="bg-cream rounded-diagonal {{ {s: 4} | spacingClasses({padding: ['top', 'bottom', 'left', 'right']}) }}">
+      <div class="{{ {s:'HNM4'} | fontClasses }}">
+        {{ event.format.title }}
+      </div>
 
-  {# Cost/type #}
-  <span>Free</span>
-  {{ ' / Drop in' if event.isDropIn }}
-  {{ ' / Booking in advance' if event.bookingEnquiryTeam }}
+      <div class="{{- [
+        {s:'WB5'} | fontClasses,
+        {s: 2} | spacingClasses({margin: ['top']}),
+        {s: 1} | spacingClasses({margin: ['bottom']})
+      ].join(' ') -}}">
```

Comment:
neat trick!

Reply:
Yea, I was surprised it worked.
Keeps the markup that wee bit tinier...